src/clients/api/queries/getMintableVai/index.ts

```diff
@@ -0,0 +1,15 @@
+import { GetMintableVaiInput, GetMintableVaiOutput } from './types';
+import formatToOutput from './formatToOutput';
+
+export * from './types';
+
+const getMintableVai = async ({
+  vaiControllerContract,
+  accountAddress,
+}: GetMintableVaiInput): Promise<GetMintableVaiOutput> => {
```

Comment:
```suggestion
interface ContractInput {
   vaiControllerContract: VaiUnitroller
}

const getMintableVai = async ({
  vaiControllerContract,
  accountAddress,
}: GetMintableVaiInput): Promise<GetMintableVaiOutput & ContractInput> => {
```

Reply:
The contract from the input should not be returned, but I think you meant to divide the input in two. I wouldn't do that, as then `GetMintableVaiInput` would be incorrect; it should contain all the input parameters of the function, including the contract itself.

Reply:
yeah I miswrote it. Updated

Reply:
Got you. I still wouldn't divide the input as it can lead to misunderstandings for future developers: if one imports `GetMintableVaiInput`, you'd expect it contains all the parameters required by `getMintableVai`